src/Test/L0/Worker/TestResults/XUnitResultReaderTests.cs

```diff
@@ -320,23 +549,23 @@ public void VerifyLastTestCaseEndDateNotGreaterThanTestRunTotalTime()
         [Fact]
         [Trait("Level", "L0")]
         [Trait("Category", "PublishTestResults")]
-        public void VerifyStartDateIsEmptyWhenNoRunTimeIsAvailable()
+        public void VerifyStartDateIsNotEmpty()
```

Comment:
There is behavior change here, now start date and end date would be never empty. Hence the tests were modified.